CMakeLists.txt

```diff
@@ -115,6 +115,10 @@ find_program( SED NAMES sed gsed )
 ##################   CPack, checks, ... in external modules   ##################
 ################################################################################
 
+include( GetTriple )
```

Comment:
I don't like the name `GetTriple` for this file. Can you make it more specific, please?`Maybe `DetermineArchitecture` or something like this.